lib/SimpleSAML/Memcache.php

```diff
@@ -277,7 +290,12 @@ private static function addMemcacheServer($memcache, $server)
         }
 
         // add this server to the Memcache object
-        $memcache->addServer($hostname, $port, true, $weight, $timeout, $timeout, true);
+        if (self::$extension == 'memcached') {
+            $memcache->addServer($hostname, $port);
+        }
+        else {
+            $memcache->addServer($hostname, $port, TRUE, $weight, $timeout, $timeout, TRUE);
```

Comment:
We are using the [PSR-2](http://www.php-fig.org/psr/psr-2/) coding standard, which mandates lowercase in all keywords, including `true`, `false` and `null`. The else statement should also be in the same line as the closing brace of the if.